src/pymonzo/transactions/schemas.py

```diff
@@ -162,6 +197,7 @@ class MonzoTransaction(BaseModel):
     description: str
     id: str
     merchant: Union[MonzoTransactionMerchant, str, None]
+    counterparty: Union[MonzoTransactionCounterparty, dict, None] = None
```

Comment:
Do you mind moving this below `settled` please, alongside other attributes with default values? Thanks!

Also, should we accept `dict` here? Shouldn't be either in the format specified in `MonzoTransactionCounterparty` or `None`?

Reply:
I think I've cleaned this up now...